packages/tools/devtools/devtools-core/package.json

```diff
@@ -156,15 +156,8 @@
 		"rimraf": "^4.4.0",
 		"typescript": "~5.4.5"
 	},
-	"fluidBuild": {
-		"tasks": {
-			"tsc": [
-				"typetests:gen",
-				"build:genver"
```

Comment:
build:genver dependency is stale? (bonus cleanup?)

Reply:
Yes, none of the task overrides are needed.